src/components/editMeasure/editor/MeasureEditor.tsx

```diff
@@ -540,6 +540,22 @@ const MeasureEditor = () => {
     return result.status;
   };
 
+  const handleParameterEdit = (
```

Comment:
we can probably addd a unit test for this

Reply:
@adongare Just added a test for it, thanks Ashok!